### ISSUE TITLE
[a11y] `Use saved card` and `done` buttons cannot be reached with the switch control while Voice-over is ON

### ISSUE THREAD

**Issue by abodea** [mozilla-mobile/firefox-ios#16160]
## Steps to reproduce
1. Turn on Voice Over.
1. Access any checkout page.
1. Attempt to reach `use saved card` or `done` buttons with the switch control.
### Expected behavior
The `use saved card` or `done` buttons should be reachable with the switch control.
### Actual behavior
`Use saved card` and `done` buttons cannot be reached with the switch control while Voice-over is ON
### Device & build information

* Device: iPhone 13 Pro (15.7.1). 
* Build version: v117 (33597) 
* First seen version: N/A 
### Notes
Attachments: 

![251031990-ed759c56-4971-44ab-a418-c6dc75dfbf65](https://github.com/mozilla-mobile/firefox-ios/assets/42831109/67580b00-a8be-4755-b0a7-9f695a9d7270)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-7280)

**Comment by lmarceau** [comment]
@thatswinnie this might be picked up by a contributor, but wondering if you know how to make this top of the keyboard section accessible?

**Comment by thatswinnie** [comment]
@lmarceau unfortunately I don't know that. I don't think it will be different than other views but to be save maybe an investigation would be good.

**Comment by imsanchit** [comment]
hey @thatswinnie, could we please somehow check if other keyboard actions like toolbar buttons (up-down arrow) or keys are accessible via switch control?

**Comment by thatswinnie** [comment]
@imsanchit in Firefox for iOS I was unfortunately unable to reach any accessory view action using switch control. In Safari that was possible however.

**Comment by donalddang** [comment]
It looks like this was addressed in Feb 2024 based on the commit history.
